OneSignalSDK/onesignal/core/src/main/java/com/onesignal/core/internal/operations/impl/OperationRepo.kt

```diff
@@ -352,4 +360,16 @@ internal class OperationRepo(
 
         return ops
     }
+
+    /**
+     * Load saved operations from preference service and add them into the queue
+     * NOTE: sometimes the loading might take longer than expectedly due to device's state
+     */
+    private fun loadSavedOperations() {
+        // load operation in a separate thread to avoid halting the main process
```

Comment:
Would call out due to disk I/O here, or we probably can just remove this comment, since it will be duplicate with method as a whole comment above this one.